oomph_build.py

```diff
@@ -506,7 +506,7 @@ def expanded_path(p):
     oomph_group.add_argument("--oomph-OOMPH_EXTRA_COMPILE_DEFINES", metavar="DEFINES", help="Additional C/C++ compile defines to pass to the oomph-lib build.")
 
     # Any additional flags for oomph-lib
-    oomph_group.add_argument("--oomph-extra-flags", nargs="+", metavar="FLAG", help="Additional raw CMake flags for oomph-lib (e.g. -DXYZ=VALUE).")
+    oomph_group.add_argument("--oomph-extra-cmake-options", nargs="+", metavar="FLAG", help="Additional raw CMake flags for oomph-lib (e.g. -DXYZ=VALUE).")
```

Comment:
DUPLICATED COMMENT: I always struggle with these overly generic placeholders. Can we specify a real example of an option one may actually want to pass to cmake; ideally something that users are likely to have come across already and will immediately recognise as something that would only be passed to cmake.